src/libs/customEvents.js

```diff
@@ -0,0 +1,25 @@
+
+function fire(name, params, ...additionalParams) {
```

Comment:
I think we can remove `additionalParams`, I don't see why `params` would not be enough

Reply:
Indeed. I just copied the previous code but this is worth simplifying. Thank you!